### DISCUSSION TITLE
Add support for SPAWNERMETA

### DISCUSSION THREAD

**Discussion by ghost** [BG-Software-LLC/SuperiorSkyblock2#842] [Denied Feature Requests] [answered]
Hello ! 
Can you add support for : https://www.spigotmc.org/resources/spawnermeta-upgrade-and-customize-spawners-1-14-1-18.74188/

Ty :D

**Comment by OmerBenGera** [comment, marked as answer]
SpawnerMeta can add support for SSB on their own and register their own block keys using the API.
Here's a tutorial on how they can do it:
https://wiki.bg-software.com/superiorskyblock/api#register-your-own-block-keys
